seed/data_importer/tasks.py

```diff
@@ -954,223 +817,21 @@ def filter_duplicated_states(unmatched_states):
     :return:
     """
 
+    # TODO #239: Should we save the hash in the database, wouldn't that be faster
```

Comment:
+1

Reply:
this will happen in the performance branch. Thanks for your help with this one. It was a doozy, but I think it is much nicer.